build.sh

```diff
@@ -66,6 +66,7 @@ if [ ! -d /app ]; then
 		unset GIT_REPO
 		cd /src
 		git checkout $GIT_TAG
+		GIT_SHA1=$(git rev-list $GIT_TAG | head -n 1)
```

Comment:
could you do `export GIT_SHA1=` so the env var is available to subprocesses of the script?